plugins/premium/tinymcespellchecker.md

```diff
@@ -322,3 +335,15 @@ tinymce.init({
   }
 });
 ```
+
+## API
+
+| Name | Arguments | Description |
```

Comment:
Should this instead be in `_includes/plugin-apis/tinymcespellchecker-apis.md` and then included here? I thought @tylerkelly13 had mentioned a while back we should do that, as he was trying to break these up (I could be recalling incorrectly though).

Reply:
yes please. @JamesToohey, the exports plugin is a good example.